contracts/cw20-merkle-airdrop/helpers/package.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Did you build this from scratch or modify an existing project?

If it was based on other work, it would be good to check their license. MIT and BSD is more like "no what you want". Apache does require us to post notice of the prior work it builds on (give them credit/link to it). GPL is more problematic

Reply:
Built from scratch.